samples/batch/transcription-enabled-storage/Connector/Serializable/ServiceBusMessage/Data.cs

```diff
@@ -0,0 +1,32 @@
+﻿// <copyright file="Data.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace Connector
+{
+    using System;
+
+    public class Data
```

Comment:
It is not clear what Data means?

Reply:
inside of Connector/Serializable are all objects that we are deserializing from JSON/ServiceBusMessage. It's just called Data because the field inside of the Azure Service Bus Message is called Data and we just have these Objects to deserialize and use them.